helix-term/src/ui/editor.rs

```diff
@@ -163,15 +163,18 @@ impl EditorView {
             Box::new(highlights)
         };
 
-        Self::render_gutter(
-            editor,
-            doc,
-            view,
-            view.area,
-            theme,
-            is_focused,
-            &mut line_decorations,
-        );
+        let gutter_overflow = view.gutter_offset(doc) == 0;
```

Comment:
Not sure if this is the right way to go about it or if `gutter_offset` should return an `Option<u16>` instead with `None` in case of overflow